### PR TITLE
Add standard gem and autocorrect linting errors

### PR DESCRIPTION
**Pull request by gcentauri**
@Gmfholley mentioned i've got some linting violations in my PR and so I thought i'd go ahead and add in this gem

### CODE REVIEW

File: .standard_todo.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Auto generated files with errors to ignore.
```
Comment (alecdotninja) [inline .standard_todo.yml:1]: If you remove this file and run `standardrb --fix` can it automatically correct these issues? If so, I would prefer to do that now.

**Comment by gcentauri** [comment]
> Should we also add this to run on GH when the PR is pushed up?
> 
> It looks like motion is using travis. Or maybe we can use GitHub Actions.

Yeah, it would be nice. I have some tests around the Go stuff too if we want to run unit tests as well... they're currently broken since i refactored everything (game still works :P, so maybe i should say *rewrote*)

**Comment by gcentauri** [comment]
I added a github action to run linter. Seems like tests won't run here or locally for me at the moment.

**Comment by gcentauri** [comment]
@Gmfholley @alecdotninja - is it cool to merge this one?

**Pull request merged**